jormungandr/src/rpc/eth_types/block.rs

```diff
@@ -0,0 +1,157 @@
+use super::transaction::Transaction;
+use chain_evm::ethereum_types::{Bloom, H160, H256, U256};
+use serde::{Serialize, Serializer};
+
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub struct Bytes(Vec<u8>);
```

Comment:
`Box<[u8]>` is probably better.

PS: This could probably be a data type shared somewhere

Reply:
updated